### PR TITLE
Put "run_conformance.py" and all test runner files ".csv" in build directory

### PR DESCRIPTION
**Pull request by bhargavthriler**
Put "run_conformance.py" and all test runner files ".csv" in build directory
Added steps in CMakeLists.txt to copy the ".csv" files and "run_conformance.py" from 'test_conformance' directory

Signed-off-by: bhargavthriler <bhargavthriler@gmail.com>

### CODE REVIEW

File: test_conformance/CMakeLists.txt [hunk 1]
```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+#configure_file(../test_conformance/opencl_conformance_tests_quick.csv . COPYONLY)
```
Comment (kpet) [inline test_conformance/CMakeLists.txt]: I'm guessing you didn't intend to keep this.

File: test_conformance/CMakeLists.txt [hunk 2]
```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
...
+
+#copy all csv files
+file(GLOB CSV_FILES
+ 	 "../test_conformance/*.csv"
```
Comment (kpet) [inline test_conformance/CMakeLists.txt]: It would be better to avoid this use of relative paths.

File: test_conformance/CMakeLists.txt [hunk 3]
```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
...
+)
+FILE(COPY ${CSV_FILES} DESTINATION .)
+
+#copy test runner
+FILE(COPY "../test_conformance/run_conformance.py" DESTINATION ".")
```
Comment (kpet) [inline test_conformance/CMakeLists.txt]: Same comment here. Can the path be constructed from variables?

File: test_conformance/CMakeLists.txt [hunk 4]
```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+#configure_file(../test_conformance/opencl_conformance_tests_quick.csv . COPYONLY)
+
+#copy all csv files
```
Comment (aarongreig) [inline test_conformance/CMakeLists.txt]: For style consistency with the rest of the file could you add a space and a capital here? e.g.
`# Copy all csv files`

File: test_conformance/CMakeLists.txt [hunk 5]
```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
...
+file(GLOB CSV_FILES
+ 	 "../test_conformance/*.csv"
+)
+FILE(COPY ${CSV_FILES} DESTINATION .)
+
+#copy test runner
```
Comment (aarongreig) [inline test_conformance/CMakeLists.txt]: As above, missing space and capital

File: test_conformance/CMakeLists.txt [hunk 6]
```diff
@@ -89,3 +89,15 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+#configure_file(../test_conformance/opencl_conformance_tests_quick.csv . COPYONLY)
+
+#copy all csv files
+file(GLOB CSV_FILES
```
Comment (aarongreig) [inline test_conformance/CMakeLists.txt]: Again for consistency could you either capitalize all of the `file` calls or make them all lower case?

File: test_conformance/CMakeLists.txt (hunk 1 above)
Comment (bhargavthriler) [reply test_conformance/CMakeLists.txt]: Yeah it was mistake,I'll remove that

File: test_conformance/CMakeLists.txt (hunk 2 above)
Comment (bhargavthriler) [reply test_conformance/CMakeLists.txt]: ok , I'll fix it

**Comment by bhargavthriler** [comment]
Hi @kpet @aarongreig I made the changes as you asked. Can you review the new changes.

File: test_conformance/CMakeLists.txt [hunk 7]
```diff
@@ -89,3 +89,13 @@ add_subdirectory(vec_step)
 
 # Add any extension folders
 add_subdirectory(spir)
+
+# Copy all csv files
+FILE(GLOB CSV_FILES
+ 	 "${CMAKE_CURRENT_SOURCE_DIR}/*.csv"
+)
+FILE(COPY ${CSV_FILES} DESTINATION ".")
```
Comment (bhargavthriler) [inline test_conformance/CMakeLists.txt]: @kpet  For the current working directory I couldn't any variables. I mean to specify the destination directory.

Comment (bhargavthriler) [reply test_conformance/CMakeLists.txt]: > @kpet For the current working directory I couldn't any variables. I mean to specify the destination directory.

fixed it by using "CMAKE_BINARY_DIR" variable

**Comment by bhargavthriler** [comment]
Hi @kpet  @aarongreig can this PR be merged ?

**Pull request merged**